p2p/pex/pex_reactor.go

```diff
@@ -658,7 +664,15 @@ func (r *Reactor) AttemptsToDial(addr *p2p.NetAddress) int {
 func (r *Reactor) crawlPeersRoutine() {
 	// If we have any seed nodes, consult them first
 	if len(r.seedAddrs) > 0 {
-		r.dialSeeds()
+		for try := 0; try < seedInitMaxAttemptToDial; try++ {
+			// we sleep a few (random) secs to avoid connection storm when whole network restarts
+			delay := time.Duration(tmrand.Int63n(seedConnectMaxDelayPeriod.Nanoseconds()))
```

Comment:
1. What do you think about to move `sleep` logic into separate function, like `randSleep`?
2. the function `Int63n` returns a value in the interval [0,n), what do you think about to use (m, n) interval? `rand.Int63n(m - n) + m`

Reply:
Ad 1 - I see a need for a bigger refactoring of this code, but upstream is already refactoring p2p package, so I'd like to avoid any unnecessary changes.

Ad 2 - 0..n is good enough. On average it will retry every 2.5s * 12 retries = 30 s of attempts (assuming the connection attempt itself takes nothing). 

If we want to do this correctly, we should follow logic from `dialPeer()` https://github.com/dashevo/tenderdash/blob/7a8044c8835a2f4f078759e81495908aafd2d334/p2p/pex/pex_reactor.go#L556-L559 
but it involves much more refactoring than I'm willing to do.